build/configs/rtl8730e/loadable_ext/defconfig

```diff
@@ -501,7 +501,12 @@ CONFIG_SPI_EXCHANGE=y
 
 #
 # LCD Driver Support
-#
+CONFIG_LCD=y
+CONFIG_LCD_ST7785=y
+CONFIG_AMEBASMART_MIPI=y
+CONFIG_MIPI_DSI=y
+CONFIG_EXAMPLES_LCD=y
```

Comment:
You add them manually. We should not.
Even you add XRES and YRES configs, here is no value.

Please modify the defconfig using menuconfig.
After modifying it using menuconfig, and then copy .config to defconfig.